src/components/Status.js

```diff
@@ -0,0 +1,146 @@
+import React from 'react';
+import {
+  SprkLink,
+  SprkIcon,
+  SprkTooltip
+} from '@sparkdesignsystem/spark-react';
+
+const Status = ({
+    statusName,
+    href,
+    additionalInfo
+}) => {
+    if (statusName === "ready") {
+        return (
+            <div>
+                <SprkIcon className="
+                    docs-status__Ready 
+                    sprk-c-Icon--filled
+                    sprk-c-Icon--l"
+                    iconName="check-mark-filled"
+                />
```

Comment:
The icon for each of these statuses should have `aria-hidden="true"` so they are hidden from screen readers and other assistive technology. We don't need to provide a label for these images (like with `aria-label`) because the link text immediately following this image adequately describes the link.